app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_type/AddBusStopType.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.bus_stop_type;
+
+import android.os.Bundle;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.OverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopType extends OverpassQuestType
+{
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes with public_transport=platform and !shelter and !informal";
```

Comment:
I think the questions should be as atomic as possible. I'd advertise for a simple yes/no answer. Then, have two quests (with same icon, same layout): bench? yes/no, shelter? yes/no

Here are some arguments for a yes/no answer:
- User can answer them with **one** click (Instead of OK button, there'd be a Yes and a No button). With images, you have to at least click once to select the image, then click Ok. With 4 images and more, you'll probably even have to swipe down before you can click Ok.
- Only **one** string needs to be translated, the question itself (currently translation is maintained in 16 languages and growing)
- The images carry no information value. Everybody knows what a bench is and what a shelter is. For roof shapes and surface, images make sense because some people might not know what a "saltbox" roof is and what a "sett" surface is
- Simple Yes-No quest forms can be made into a generic fragment that can be reused by many other simple yes-no quests (i.e.: bus stop shelter?, bus stop bench?, street lit?, parking space requires fee?, toilet requires fee? etc. etc.), thus -> less work